### PR TITLE
Update XPMEM to work with Linux kernels up to v5.3

### PR DESCRIPTION
**Pull request by jdinan**
Tested with Ubuntu LTS running Linux v5.0 and v5.3 kernels.

### CODE REVIEW

**Comment by jdinan** [comment]
@hjelmn Your fork seems to be the common source for XPMEM. Are you accepting PRs? 😀

**Comment by jdinan** [comment]
@hjelmn I added a build/test action to the PR to build and test the kernel module. You may need to enable the corresponding github action (or adding me as a collaborator may work, too) for the testing to run. Works great on my fork https://github.com/jdinan/xpmem/pull/2.

File: kernel/xpmem_mmu_notifier.c [hunk 1]
```diff
@@ -65,10 +65,20 @@ xpmem_invalidate_PTEs_range(struct xpmem_thread_group *seg_tg,
  * XPMEM only uses the invalidate_range_end() portion. That is, when all pages
  * in the range have been unmapped and the pages have been freed by the VM.
  */
+#if LINUX_VERSION_CODE < KERNEL_VERSION(5, 0, 0)
```
Comment (shamisp) [inline kernel/xpmem_mmu_notifier.c]: So, this 5.0.0 thing and not 5.3.0 ? just to confirm.

**Comment by shamisp** [comment]
👍

**Comment by hjelmn** [comment]
Yes. I accept PRs. Thanks!

File: kernel/xpmem_mmu_notifier.c (hunk 1 above)
Comment (jdinan) [reply kernel/xpmem_mmu_notifier.c]: Yes, for each change I looked back through the Linux releases to determine when the change was first introduced. I tested the patch on Ubuntu LTS with Linux 5.0.0-37-generic and 5.3.0-26-generic.

**Pull request merged**